src/platform/ios/mod.rs

```diff
@@ -150,6 +150,23 @@ impl MonitorId {
     pub fn get_hidpi_factor(&self) -> f32 {
         1.0
     }
+
+    pub fn get_physical_extents(&self) -> Option<(u64, u64)> {
+        unsafe {
+            if setjmp(mem::transmute(&mut jmpbuf)) != 0 {
+                let app: id = msg_send![Class::get("UIApplication")?, sharedApplication];
+                let delegate: id = msg_send![app, delegate];
+                let state: *mut c_void = *(&*delegate).get_ivar("glutinState");
+                let state = state as *mut DelegateState;
+                let scale = (*state).scale;
+                let width = (*state).size.0;
+                let height = (*state).size.1;
+                Some(((width as f32 * scale) as u64, (height as f32 * scale) as u64))
```

Comment:
and note that converting a float to an integer doesn't automatically round. Is this what you wanted?